checksec

```diff
@@ -683,27 +686,43 @@ if [ "$arch" == "arm" ]; then
 fi
 
   echo_message "\n" "\n" "\n" ""
-  echo_message "* Selinux:             " "" "" ""
+  echo_message "* SELinux:                                " "" "" ""
   if $kconfig | grep -qi 'CONFIG_SECURITY_SELINUX=y'; then
   getsestatus
   sestatus=$?
-  if [ $sestatus == 0 ]; then 
-      echo_message "\033[31mDisabled\033[m\n" "Disabled" "    <selinux enabled='no' />" '"selinux":{ "enabled":"no" },'
+  if [ $sestatus == 0 ]; then
+      echo_message "\033[31mDisabled\033[m\n" "Disabled,," "    <selinux enabled='no'" '"selinux":{ "enabled":"no"'
```

Comment:
i adopted the behavior from https://github.com/slimm609/checksec.sh/blob/master/checksec#L932, so that there always the same number of commas
but i might misinterpret the format